go.mod

```diff
@@ -277,3 +277,5 @@ replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 replace github.com/gruntwork-io/terragrunt => github.com/infracost/terragrunt v0.47.1-0.20240501143558-4c01e72103df
 
 replace github.com/heimdalr/dag => github.com/aliscott/dag v1.3.2-0.20231115114512-4ce18c825f94
+
+replace github.com/shurcooL/graphql => github.com/spacelift-io/graphql v1.2.0
```

Comment:
I've updated this as the spacelift client requires their internal fork, the fork adds some additional request options. There seems to be no significant divergance from the base library and considering https://github.com/shurcooL/graphql is stable I think this change is fine